### DISCUSSION TITLE
不是吧哥们，你这让我怎么活啊

### DISCUSSION THREAD

**Discussion by cnbilinyj** [Jeric-X/SyncClipboard#104] [General]
![image](https://github.com/user-attachments/assets/d0a3410f-2e4a-492d-81e4-2d3c69e30c0e)
不是本来我就是想着在Auto.js Pro或者Autox.js里面整个脚本，然后电脑端整个Python，结果我发现找不到怎么用Auto.js Pro或者Autox.js创建WebSocket服务器，然后好奇看看网络上面有没有这种东西，就是那种已经做完的，结果看到你这，好家伙，我原本以为只是单纯的电脑端和手机端的软件互通，结果，你倒好，把我要用的工具都给用了，我用的工具跑的脚本你都帮我写好了，不是吧哥们你这让我怎么我怎么活啊？我想着用Auto.js Pro或者Autox.js，结果你倒好。我以为只是手机端还要在装个软件，没想到啊，你倒好，直接给写了个js的脚本，适用于Autox.js或者也包括Auto.js Pro的JavaScript脚本，不是吧你这让我怎么么活啊……

**Comment by cnbilinyj** [comment]
算了脚本下载下来我看看吧

**Comment by xtccc** [comment]
使用http request呗 ，这个挺好用的
